packages/clay-css/src/scss/variables/_tables.scss

```diff
@@ -391,6 +391,7 @@ $c-tr-table-focus: () !default;
 $c-tr-table-focus: map-deep-merge(
 	(
```

Comment:
I think we need to add `outline: 0` here. I'm seeing a double border for `tr` focus.
![double-border](https://github.com/liferay/clay/assets/788266/6eb26a5c-8313-4525-a881-0dc97b049c87)